dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/netty/message/NettyResponseMessage.java

```diff
@@ -45,6 +45,7 @@
 import com.ibm.wsspi.http.channel.values.VersionValues;
```

Comment:
In my opinion there are enough changes here to add a date range to the copyright header that includes 2025